scripts/us_bls/jolts/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+    "import_specifications": [
+        {
+            "import_name": "BLS_JOLTS",
+            "curator_emails": [
+                "ianjc@google.com"
+            ],
+            "provenance_url": "https://www.bls.gov/jlt/",
+            "provenance_description": "U.S. Bureau of Labor Statistics data on job openings, hires, and separations.",
+            "scripts": [
+                "bls_jolts.py"
+            ],
+            "import_inputs": [
+                {
+                    "template_mcf": "BLSJolts.tmcf",
+                    "cleaned_csv": "BLSJolts.csv",
+                    "node_mcf": "BLSJolts_StatisticalVariables.mcf"
+                }
+            ],
+            "cron_schedule": "0 10 17 * *"
```

Comment:
Did we choose this to match the typical release dates?

Reply:
It seems like there isn't a set release date. We can also make it the last day of a month

Reply:
@IanCostello I defer to Ian's opinion.

Reply:
It seems to always come out by the 19th if I recall. By the end of the month could be a good option. Or on the 21st and the 31st to be safe.

Reply:
changed to 21st